dev-requirements.txt

```diff
@@ -1,7 +1,7 @@
 # install latest changes in dbt-core
 # TODO: how to automate switching from develop to version branches?
-git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-core&subdirectory=core
-git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-tests-adapter&subdirectory=tests/adapter
+git+https://github.com/dbt-labs/dbt-core.git@er/ct-1687-rename-exceptions#egg=dbt-core&subdirectory=core
```

Comment:
Could you offer some of your reasoning for why we should change this reference to a specific branch? Why not address this after merging your [dbt core-PR](https://github.com/dbt-labs/dbt-core/pull/6539) into `main`? 

I love that we were able to verify a green build on this branch, but my gut tells me we're wiser to wait to merge this until the change here no longer requires changes to dev-requirements.

Reply:
100% wait to merge and this will point back at main.  I just wanted to get it up for review now instead of waiting on my `dbt-core` PR to be merged in.